erpnext/crm/custom/contract.json

```diff
@@ -0,0 +1,31 @@
+{
+ "custom_fields": [],
+ "custom_perms": [],
+ "doctype": "Contract",
+ "property_setters": [
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "creation": "2020-05-29 02:56:42.849889",
+   "default_value": null,
+   "doc_type": "Contract",
+   "docstatus": 0,
+   "doctype_or_field": "DocType",
+   "field_name": null,
+   "idx": 0,
+   "modified": "2020-05-29 02:56:42.849889",
+   "modified_by": "Administrator",
+   "name": "Contract-default_print_format",
+   "owner": "Administrator",
+   "parent": null,
+   "parentfield": null,
+   "parenttype": null,
+   "property": "default_print_format",
```

Comment:
Can this happen in the Contrac's doctype form instead?

Reply:
No this has to go as a customization, cant add default print format for standard doctypes